backend/accessibility_templates/aa-004.xml

```diff
@@ -0,0 +1,21 @@
+<AccessibilityAttributes>
+ <SubtitleAttributes>
+  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023::3"/>
+  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:4"/>
+  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:99"/>
+  <SubtitleLanguage>zh-hans</SubtitleLanguage>
+  <SuitableForTTS>false</SuitableForTTS>
+ </SubtitleAttributes>
+ <AudioDescriptionAttributes>
+  <AudioAttributes>
+   <AudioLanguage>tgr</AudioLanguage>
+   <BitRate>3000</BitRate>
+  </AudioAttributes>
+  <ReceiverMix>true</ReceiverMix>
+ </AudioDescriptionAttributes>
+  <SigningAttributes>
+  <Coding href="urn:dvb:metadata:cs:VideoCodecCS:2022:5.1.10"/>
+  <SignLanguage>sgn</SignLanguage>
+  <Closed>true</Closed>
```

Comment:
Set closed to false as this functionality is usually provided using an app. It is debatable if a second stream with a burnt-in signlanguage interpreter could be used for a "closed" service, but this depends very much on the UI and as long this is not provided through an app I would argue that the service is not closed.

Reply:
OK